dist/Dockerfile.e2e/Dockerfile

```diff
@@ -1,4 +1,31 @@
-FROM quay.io/app-sre/cincinnati:builder AS rust_builder
+FROM registry.centos.org/centos/centos:8 as rust_builder
+
+# base: EPEL repo for extra tools
+RUN yum -y install epel-release
+
+# build: system utilities and libraries
+RUN yum update -y && \
+    yum -y install gcc openssl-devel jq skopeo buildah && \
```

Comment:
`/usr/bin/cincinnati-e2e-test: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory`

I think we want `openssl`, not `-devel` here

Reply:
updated

Reply:
Wrong container, these are installed in `builder`, not in resulting ubi8